### PR TITLE
docs: include ics4Wrapper in `hostkeeper.NewKeeper` ics27 integration docs 

### PR DESCRIPTION
**Pull request by damiannolan**
## Description

- Updating ics27 integration docs based on #2302 

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by chatton — APPROVED** [review]

File: docs/apps/interchain-accounts/integration.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ app.ICAControllerKeeper = icacontrollerkeeper.NewKeeper(
 )
 app.ICAHostKeeper = icahostkeeper.NewKeeper(
 		appCodec, keys[icahosttypes.StoreKey], app.GetSubspace(icahosttypes.SubModuleName),
+        app.IBCKeeper.ChannelKeeper, // may be replaced with middleware such as ics29 fee
```
Comment (crodriguezvega) [inline docs/apps/interchain-accounts/integration.md:83]: Alignment police! 🚓

Comment (damiannolan) [reply docs/apps/interchain-accounts/integration.md:83]: ahh! tabs vs spaces 🤯 

lemme fix! :)

Comment (damiannolan) [reply docs/apps/interchain-accounts/integration.md:83]:
```suggestion
		app.IBCKeeper.ChannelKeeper, // may be replaced with middleware such as ics29 fee
```

Comment (crodriguezvega) [reply docs/apps/interchain-accounts/integration.md]: > ahh! tabs vs spaces

The perennial battle! ⚔️

**Review by crodriguezvega — APPROVED** [review]

**Pull request merged**